_posts/2020-11-09-Basics-With-Bioawk.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Basics with Bioawk!"
+date:   2020-11-09 4:45:00 -0800
+categories: Bioawk
+---
+
+# Basics with ~Babish~ Bioawk!
+by Sahil Verma
+
+## Taking the Awk out of Bioawk
+
+Awk is a Unix/Linux coding language that allows coders to extract and manipulate data and generate reports based on the data. While it is a full fledged scripting language, it isn't as powerful as other languages, such as Python. Awk allows people to do data-processing tasks on tabular data. Awk does this by working on _records_ and _fields_. Since Awk works on tabular data, Awk views _records_ as rows and _fields_ as the columns for each record. In Awk, the entire record is stored to `$0`, the field's first value is assigned to `$1`, the second value is assigned to `$2`, and the pattern continues.
+
+Awk programs are, generally, built using the following recipe:
+
+`pattern {action}`
+
+The _pattern_ is an expression or regular expression patter, allowing people to filter for matched expressions, so that those only those patterns are acted upon. These _pattern-actions_ can be linked together into a chain by semicolons. By combining these two different concepts, _records_ and _fields_ with _pattern-actions_, Awk is able survey a file line by line and split those lines into fields, compare those lines and fields to a pattern, and perform actions on those patters to manipulate data and generate reports.
```

Comment:
I think maybe you forgot a word here as well. You write "Awk is able survey ...", do you mean to put "Awk is able to survey ..." ?